.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
You should add it in the .gitignore or in your local gitignore .git/info/exclude